packages/smart-wallet/src/smartWallet.js

```diff
@@ -238,10 +295,15 @@ export const prepareSmartWallet = (baggage, shared) => {
     };
   };
 
+  const StoreShape = M.remotable();
```

Comment:
imo the only reason to define this instead of inlining `M.remotable()` is to that eventually we can annotate a static type with the kind of remotable. 

consider MapStoreShape since it's not just any store.

Reply:
Thanks for the reminder. I meant to look further into a better shape. A label seems useful: `M.remotable('MapStore')`

Reply:
p.s. no longer needed